unity-environment/Assets/ML-Agents/Examples/SharedAssets/Scripts/GroundContact.cs

```diff
@@ -12,16 +12,15 @@ namespace  MLAgents
     [DisallowMultipleComponent]
     public class GroundContact : MonoBehaviour
     {
-        // [HideInInspector]
+        [HideInInspector]
         public Agent agent;
 
         [Header("Ground Check")]
-        public bool agentDoneOnGroundContact; //reset agent on ground contact?
-        public bool penalizeGroundContact; //if this body part touches the ground should the agent be penalized?
-        public float groundContactPenalty; //penalty amount (ex: -1)
+        public bool agentDoneOnGroundContact; // Reset agent on ground contact?
```

Comment:
I think question marks should be avoided in comments. 
We need to standardize our comments.

Reply:
Removed it.